src/main/java/training/endava/skeleton/Person.java

```diff
@@ -0,0 +1,61 @@
+package training.endava.skeleton;
+
+public class Person implements SkeletonObject<Person, Integer> {
+    private Integer id;
+    private String name;
+    private String phoneNumber;
+    private String address;
+
+
+    public Person(Integer id,String name, String phoneNumber,String address){
+
+        this.id=id;
+        this.name=name;
+        this.phoneNumber=phoneNumber;
+        this.address=address;
+    }
+    public String getAddress(){
+
+        return address;
+    }
+    public Integer getId(){
+
+        return id;
+    }
+    public String getName(){
+
+        return this.name;
+    }
+
+    public String getPhoneNumber(){
+
+        return this.phoneNumber;
+    }
+
+    public void setId(Integer id){
+    this.id=id;
+    }
+
+    public void setName(String Name){
+        this.name=Name;
+
+    }
+    public void setPhoneNumber(String PhoneNumber){
+        this.phoneNumber=PhoneNumber;
```

Comment:
It is a common convention to use name starting with capital letter of classes (class Person) and lower case letter for variables (phoneNumber).